### PR TITLE
Re-added duplicates of vanilla IDs

### PR DESCRIPTION
**Pull request by ppebb**
There are also a few assets here

### CODE REVIEW

**Comment by ppebb** [comment]
By the way, in my second commit, all of the changes from "Finished 1/4 of Main" have been removed.

File: patches/Avalon/Terraria/GameContent/TextureAssets.Avalon.cs [hunk 1]
```diff
@@ -7,5 +7,28 @@ partial class TextureAssets
...
+		public static Asset<Texture2D> Mana3;
+		public static Asset<Texture2D> Mana4;
+		public static Asset<Texture2D> Mana5;
+		public static Asset<Texture2D> Mana6;
+		public static Asset<Texture2D> IckyCactus;
+		public static Asset<Texture2D> ChainTTex;
+		public static Asset<Texture2D> ConnectTTex;
+		public static Asset<Texture2D> OuterTTex;
+		public static Asset<Texture2D> WallofSteel;
+		public static Asset<Texture2D>[] JukeTexture = new Asset<Texture2D>[34];
```
Comment (steviegt6) [inline patches/Avalon/Terraria/GameContent/TextureAssets.Avalon.cs]: What happened to the indentation?

Comment (ppebb) [reply patches/Avalon/Terraria/GameContent/TextureAssets.Avalon.cs]: ¯\_(ツ)_/¯

File: patches/Avalon/Terraria/GameContent/TextureAssets.Avalon.cs [hunk 2]
```diff
@@ -5,8 +5,8 @@ namespace Terraria.GameContent
 {
 	partial class TextureAssets
 	{
-        public static Asset<Texture2D> Lightning1;
-        public static Asset<Texture2D> Lightning2;
+		public static Asset<Texture2D> Lightning1;
+		public static Asset<Texture2D> Lightning2;
```
Comment (steviegt6) [inline patches/Avalon/Terraria/GameContent/TextureAssets.Avalon.cs]: Fix your indentation, not mine.

**Review by steviegt6 — CHANGES_REQUESTED** [review]

**Review by steviegt6 — APPROVED** [review]

**Pull request merged**